src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
Have you got an example of how you're using the dynamic which is going to be passed back? I understand that from a description type we then can get a list of values which we can then push in to a string format to create a friendly description. Just trying to figure how how we can implement this in a more elegant way which is obvious to use from a consumer.